### PR TITLE
Remove use_ubridge flag

### PR DESCRIPTION
**Pull request by noplay**
Ref #1115

The PR is bigger than expected, because test was not running with use_ubridge enable

### CODE REVIEW

File: tests/controller/test_project_open.py [hunk 1]
```diff
@@ -105,7 +105,6 @@ def demo_topology():
                     "node_id": "64ba8408-afbf-4b66-9cdd-1fd854427478",
                     "node_type": "vpcs",
                     "properties": {
-                        "startup_script": "",
```
Comment (noplay) [inline tests/controller/test_project_open.py]: Not related to the PR

File: tests/handlers/api/compute/test_vpcs.py [hunk 2]
```diff
@@ -83,28 +84,19 @@ def test_vpcs_nio_update_udp(http_compute, vm):
                                     "rhost": "127.0.0.1",
                                     "filters": {}},
                                 example=True)
-    assert response.status == 201
+    assert response.status == 201, response.body.decode("utf-8")
```
Comment (noplay) [inline tests/handlers/api/compute/test_vpcs.py]: Adding the response body allow when the test fail to see the content of the body to debug

File: tests/conftest.py [hunk 3]
```diff
@@ -212,6 +212,7 @@ def run_around_tests(monkeypatch, port_manager, controller, config):
     config.set("VirtualBox", "vboxmanage_path", tmppath)
     config.set("VPCS", "vpcs_path", tmppath)
     config.set("VMware", "vmrun_path", tmppath)
+    config.set("Dynamips", "dynamips_path", tmppath)
```
Comment (noplay) [inline tests/conftest.py]: Not related to the PR. Just prevent dynamips binary to be called from the test

**Comment by grossmj** [comment]
I get an error with this PR. Steps to reproduce:

* Drop 2 VPCS nodes.
* Start the first one.
* Add a link between the 2 nodes.
* Start the second one.

vpcs_error

**Comment by grossmj** [comment]
Same issue with 2 Qemu nodes.

**Comment by grossmj** [comment]
Linking 2 VirtualBox VMs together: ```Error while creating link: Adapter 0 doesn't exist on VirtualBox VM 'LinuxTinycore3.8.2X-1'```

**Comment by grossmj** [comment]
I am concerned this PR breaks something we won't notice. Should we postpone it after 2.1?

**Comment by noplay** [comment]
I think we need to understand why it's fail. Because with filter we introduce more ubridge usage we need to make sure the code around ubridge is reliable.

**Comment by noplay** [comment]
I think I got the problem. Half from the PR, half existing buggy code (VirtualBox).

We have two variable ubridge and use_ubridge and sometimes we have confusion between them.

**Comment by noplay** [comment]
Sound good now

**Pull request merged**